PersonApi/V1/Logging/LogCallAspect.cs

```diff
@@ -0,0 +1,41 @@
+using AspectInjector.Broker;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+
+namespace PersonApi.V1.Logging
+{
+    [Aspect(Scope.PerInstance, Factory = typeof(LogCallAspectServices))]
```

Comment:
If we use Lambda.Logger() do we need this?

Reply:
Yes. This is concerned with automatically adding method start and and logging, regardless of logging framework, with a simple method attribute.